src/PerformanceTests/Utilities/ReferenceAssemblies/ReferenceAssemblies.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <NonShipping>true</NonShipping>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Shall it be defined as described by @sharwell in https://github.com/dotnet/arcade/issues/7132#issue-837193193?

Reply:
I don't think so.  @sharwell is suggesting a change to arcade to make this easier for repos.